app/routes/$repo.tsx

```diff
@@ -15,25 +19,32 @@ type LoaderData = {
 export const loader: LoaderFunction = async ({
   params,
 }): Promise<LoaderData> => {
-  invariant(params.repo, `expected repo param`)
+  const content = params.repo
 
-  temporarilyRedirectToComingSoon()
```

Comment:
should I keep this line `temporarilyRedirectToComingSoon()`  ? Not sure if we are going to push to prod as we continue developing